src/components/mobile/primitives/__tests__/useSheetDrag.test.tsx

```
@@ -0,0 +1,336 @@
+}: { rec: Recorded } & Partial<
+  Omit<SheetDragConfig, 'panelRef' | 'onMove' | 'onEnd' | 'onCancel'>
+>) {
+  const panelRef = useRef<HTMLDivElement>(null);
+  const { handlers, consumeDragClick } = useSheetDrag({
+    panelRef,
+    onMove: (dy, h) => rec.moves.push([dy, h]),
+    onEnd: (dy, dt) => {
+      rec.ends.push([dy, dt]);
+      return rec.claim;
+    },
+    onCancel: () => (rec.cancels += 1),
```

ESLint

[error] Error: This value cannot be modified

Modifying component props or hook arguments is not allowed. Consider using a local variable instead.

/home/jailuser/git/src/components/mobile/primitives/__tests__/useSheetDrag.test.tsx:39:22
  37 |       return rec.claim;
  38 |     },
> 39 |     onCancel: () => (rec.cancels += 1),
     |                      ^^^ `rec` cannot be modified
  40 |     ...cfg,
  41 |   });
  42 |   return (

(react-hooks/immutability)